rules/linux/defense_evasion_log_files_deleted.toml

```diff
@@ -0,0 +1,44 @@
+[metadata]
+creation_date = "2020/11/03"
+maturity = "production"
+updated_date = "2020/11/03"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies deletion of Linux sensitive system logs. This may indicate an attempt to evade detection or destroy forensic
+evidence on a system.
```

Comment:
```suggestion
Identifies the deletion of sensitive Linux system logs. This may indicate an attempt to evade detection or destroy forensic
evidence on a system.
```

Reply:
re-lint